polling_stations/apps/councils/models.py

```diff
@@ -34,3 +36,45 @@ def __str__(self):
 
     class Meta:
         ordering = ("name",)
+
+    @property
+    def nation(self):
+        nations_lookup = {
+            "E": "England",
+            "W": "Wales",
+            "S": "Scotland",
+            "N": "Northern Ireland",
+        }
+        # A GSS code is:
+        #   'ANN' + 'NNNNNN' where 'A' is one of 'ENSW' and 'N' is 0-9.
+        #   'ANN' section is the Entity Code. The entity code's we're interested in are
+        # We want to capture gss codes for organisations who manage elections
+
+        # Ref: https://geoportal.statistics.gov.uk/search?collection=Dataset&sort=-created&tags=all(PRD_RGC)
+        gss_pattern = re.compile(
+            """
+            ^        # Start of string
+            (        # Entity Codes:
+                E06   # Unitary Authorities (England)
+              | E07   # Non-metropolitan Districts (England)
+              | E08   # Metropolitan Districts (England)
+              | E09   # London Boroughs (England)
+              | N09   # Local Government Districts (Northern Ireland)
+              | S12   # Council Areas (Scotland)
+              | W06   # Unitary Authorities (Wales)
```

Comment:
Do we know if there are any other types of entities that run elections?
@pmk01 @symroe

Reply:
Nope, that's it!